api.graphql

```diff
@@ -22,3 +25,50 @@ type PluginReturn {
 	plugin: String!
 	errors: [Status!]!
 }
+
+type PluginMasterReturn  {
+	live_site: LiveSiteConfigType
+	slack: SlackConfigType
+	twilio: TwilioConfigType
+	twitter: TwitterConfigType
+}
+
+type MessageReturn {
+    message: String
+    config: MetaDataType!
+    _id: ID!
+    createdAt: String!
+    updatedAt: String!
```

Comment:
don't think we need `updatedAt` here or in db, since these logs are written once